ros1_foxglove_bridge/src/ros1_foxglove_bridge_nodelet.cpp

```diff
@@ -26,7 +26,7 @@ constexpr uint32_t SUBSCRIPTION_QUEUE_LENGTH = 10;
 
 using ServerType = foxglove::Server<foxglove::WebSocketNoTls>;
 using TopicAndDatatype = std::pair<std::string, std::string>;
-using Subscription = ros::Subscriber;
+using SubscriptionsByClient = std::map<foxglove::ConnHandle, ros::Subscriber, std::owner_less<>>;
```

Comment:
`std::unordered_map`, unless `std::map` is required in order to use `std::owner_less`?

Reply:
Yes it only works with `std::map`. I took it from https://github.com/foxglove/ros-foxglove-bridge/blob/b3faff643e5a8aec1c2592896a3609f1b4165384/foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp#L129